draft-ietf-quic-transport.md

```diff
@@ -1387,9 +1387,13 @@ packets.
 To limit ACK blocks to those that have not yet been received by the sender, the
 receiver SHOULD track which ACK frames have been acknowledged by its peer.  Once
 an ACK frame has been acknowledged, the packets it acknowledges SHOULD not be
-acknowledged again.  To handle cases where the receiver is only sending ACK
-frames, and hence will not receive acknowledgments for its packets, it MAY send
-a PING frame at most once per RTT to explicitly request acknowledgment.
+acknowledged again.
+
+A receiver that is only sending ACK frames will not receive acknowledgments for
+its packets.  Sending an occasional MAX_DATA or MAX_STREAM_DATA frame as data is
+received will ensure that acknowledgements are generated by a peer.  Otherwise,
+an endpoint MAY send a PING frame at most once per RTT to solicit
```

Comment:
This is in the old text, but it's odd to have "MAY send .. at most once per RTT." I'd suggest removing the "at most" and changing this to "MAY send ... once per RTT."